source/dagger/gameplay/ping_pong/ping_pong_main.cpp

```diff
@@ -37,7 +37,7 @@ void ping_pong::CreatePingPongBall(float tileSize_, ColorRGBA color_, Vector3 sp
     transform.position.z = pos_.z;
     auto& ball = reg.emplace<PingPongBall>(entity);
     ball.speed = speed_ * tileSize_;
-
+    /*if (rand() % 100 > 50)ball.speed *= -1;*///DODATO
```

Comment:
something strange here

Reply:
Sorry about that. Was trying to improvise for something else.